  LetITsnow.cpp
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(40,14): error C2672: "std::invoke": не найдена соответствующая перегруженная функция
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(47): message : выполняется компиляция ссылки на экземпляр шаблон функции "unsigned int std::thread::_Invoke<_Tuple,0,1,2>(void *) noexcept"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(47): message :         with
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(47): message :         [
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(47): message :             _Tuple=_Tuple
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(47): message :         ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(53): message : выполняется компиляция ссылки на экземпляр шаблон функции "unsigned int (__stdcall *std::thread::_Get_invoke<_Tuple,0,1,2>(std::integer_sequence<unsigned int,0,1,2>) noexcept)(void *)"
C:\Users\Rodion_PC\source\repos\LetITsnow\LetITsnow.cpp(118): message : выполняется компиляция ссылки на экземпляр шаблон функции "std::thread::thread<void(__cdecl &)(int *,int *),std::reference_wrapper<int [25][25]>,std::reference_wrapper<int [2]>,void>(_Fn,std::reference_wrapper<int [25][25]> &&,std::reference_wrapper<int [2]> &&)"
C:\Users\Rodion_PC\source\repos\LetITsnow\LetITsnow.cpp(118): message :         with
C:\Users\Rodion_PC\source\repos\LetITsnow\LetITsnow.cpp(118): message :         [
C:\Users\Rodion_PC\source\repos\LetITsnow\LetITsnow.cpp(118): message :             _Fn=void (__cdecl &)(int *,int *)
C:\Users\Rodion_PC\source\repos\LetITsnow\LetITsnow.cpp(118): message :         ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\chrono(669): message : выполняется компиляция ссылки на экземпляр класс шаблон функции "std::chrono::duration<double,std::ratio<1,1>>"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\chrono(177): message : выполняется компиляция ссылки на экземпляр класс шаблон функции "std::chrono::duration<__int64,std::nano>"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\chrono(647): message : выполняется компиляция ссылки на экземпляр класс шаблон функции "std::chrono::time_point<std::chrono::steady_clock,std::chrono::steady_clock::duration>"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(36,5): error C2893: Сбой при специализации функции-шаблона "unknown-type std::invoke(_Callable &&,_Types &&...) noexcept(<expr>)".
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\type_traits(1571): message : см. объявление "std::invoke"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(36,5): message : Со следующими аргументами шаблона:
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(36,5): message : "_Callable=void (__cdecl *)(int *,int *)"
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include\thread(36,5): message : "_Types={std::reference_wrapper<int [25][25]>, std::reference_wrapper<int [2]>}"
